src/linear_algebra/prod.lean

```diff
@@ -251,6 +251,24 @@ begin
   exact ⟨⟨x, rfl⟩, ⟨x, rfl⟩⟩
 end
 
+lemma ker_coprod_of_disjoint_range {M₂ : Type*} [add_comm_group M₂] [semimodule R M₂]
+  {M₃ : Type*} [add_comm_group M₃] [semimodule R M₃]
+  (f : M →ₗ[R] M₃) (g : M₂ →ₗ[R] M₃) (hd : disjoint f.range g.range) :
+  ker (f.coprod g) = (ker f).prod (ker g) :=
+begin
+  ext x,
+  rcases x with ⟨y, z⟩,
```

Comment:
AFAIR, you can use `ext ⟨y, z⟩`. Also, one of the inequalities hold without `hd`. Could you please add it as a separate lemma?